lib/bump/cli/resource.rb

```diff
@@ -3,8 +3,8 @@
 
 module Bump
   class CLI
-    class Parser
-      def load(content)
+    class Resource
```

Comment:
Maybe the 3rd commit message should be updated, without mention to `File`

> * Rename Parser into File and #load into #parse
> * Move from instance method to class method
> * Rename File into Resource

Reply:
Done, thanks :)